src/pages/Notifications/List/BundlePage.tsx

```diff
@@ -23,22 +23,52 @@ import { linkTo } from '../../../Routes';
 import { useGetApplicationsLazy } from '../../../services/Notifications/GetApplications';
 import { Facet } from '../../../types/Notification';
 import { BundlePageBehaviorGroupContent } from './BundlePageBehaviorGroupContent';
+import { useLocation } from 'react-router-dom';
 
 interface NotificationListBundlePageProps {
   bundle: Facet;
   bundleTabs: Facet[];
   applications: Array<Facet>;
 }
 
+enum BundleTabIndex {
+  Rhel = 0,
+  Console = 1,
+  OpenShift = 2,
+}
+
```

Comment:
we could probably map the bundle names directly to the indexes and avoid the switch

Reply:
fixed